src/prefect/tasks/aws/secrets_manager.py

```diff
@@ -59,6 +62,10 @@ def run(self, secret: str = None, credentials: str = None) -> dict:
 
         secret_string = secrets_client.get_secret_value(SecretId=secret)["SecretString"]
 
-        secret_dict = json.loads(secret_string)
+        try:
+            secret_dict = json.loads(secret_string)
+        except JSONDecodeError:
+            secret_string = secret_string.strip('{"}')
```

Comment:
Based on a quick test that I did with plaintext secrets in AWS Secrets Manager, I don't think AWS forces plaintext secrets to be in curly brackets. I think this line can be removed.
```suggestion
```